components/clientComponents/forms/Review/QuestionsAnswersList.tsx

```diff
@@ -0,0 +1,28 @@
+import { randomId } from "@lib/client/clientHelpers";
+import { SubElements } from "./SubElements";
+import { ReviewElement, ReviewItem } from "./reviewUtils";
+
+export const QuestionsAnswersList = ({
+  reviewItem,
+}: {
+  reviewItem: ReviewItem;
+}): React.ReactElement => {
+  return (
+    <dl className="my-10">
+      {Array.isArray(reviewItem.elements) &&
+        reviewItem.elements.map((reviewElement) => {
+          if (Array.isArray(reviewElement.values)) {
+            return (
+              <SubElements key={randomId()} elements={reviewElement.values as ReviewElement[]} />
```

Comment:
Why not use the review item id here vs random?

Reply:
I ran into cases where the Id between the element(s) and sub element(s) would be the same - messing up reacts "caching". Why the randomId()